testfiles/output/00266.dpx-default_XmlUnitExpectedOutput.xml

```diff
@@ -1,20 +1,20 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<fits xmlns="http://hul.harvard.edu/ois/xml/ns/fits/fits_output" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://hul.harvard.edu/ois/xml/ns/fits/fits_output http://hul.harvard.edu/ois/xml/xsd/fits/fits_output.xsd" version="1.5.2-SNAPSHOT" timestamp="3/17/22 10:21 AM">
+<fits xmlns="http://hul.harvard.edu/ois/xml/ns/fits/fits_output" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://hul.harvard.edu/ois/xml/ns/fits/fits_output http://hul.harvard.edu/ois/xml/xsd/fits/fits_output.xsd" version="1.6.0-SNAPSHOT" timestamp="1/10/23, 10:03 AM">
   <identification>
-    <identity format="Digital Picture Exchange" mimetype="image/x-dpx" toolname="FITS" toolversion="1.5.2-SNAPSHOT">
+    <identity format="Digital Picture Exchange" mimetype="image/x-dpx" toolname="FITS" toolversion="1.6.0-SNAPSHOT">
       <tool toolname="embARC" toolversion="0.2" />
-      <tool toolname="Exiftool" toolversion="12.40" />

```

Comment:
This comment applies to many of the changes below: I am surprised that reordering of JHOVE/Jypylizer output required changes such as `toolversion` numbers, `executionTime`s, XML namespace `timestamps`, etc. It seems like we used to be ignoring those values or were able to leave them unchanged for some other reason.

Can we limit the changes to these files to focus on the JHOVE/Jypylizer reordering?

Reply:
Those values are ignored by the tool that does the xml comparison. Usually, when I update the expected xml, I only change the fields that need to be updated to make the tests pass and leave all of the fields that don't matter unchanged. This is why you're seeing a bunch of version changes, etc here.

Reply:
@daveneiman : could you omit updates to the elements of the test XML that can be ignored?

Reply:
Those updates must be made by hand, which I can say from personal experience, is extremely tedious. Perhaps another option here  to make a slightly better PR would be to do a PR before this one that just updates all of the expected xml with no other changes, and then do this PR that changes the tool order. Then, there would be a smaller set of differences, at least.

Reply:
I could revert the changes and update only the jp2-related tests. However, The `<statistics>` section requires changes for all test results due to the reordering of the `jpylyzer` tool.

Reply:
@pwinckles @awoods While not ideal, it was decided in scrum this morning not to go through the process of rolling back changes, updated the 4 new jp2 test results, manually updating the statistics section of all expected test results, submitting a PR, then creating another PR with updating all the expected test results showing all new tool versions (the "ignored" xml comparisons).

Reply:
My suggestion does not require any manual changes.

1. Check out `main`
2. Do whatever you did to update all of the expected xml
3. Create a PR
4. Merge PR
5. Update the tool order
6. Do whatever you did to update all of the expected xml
7. Create a PR

Reply:
2. (above) requires **manually** updating the expected output of 90 test results. That is, doing a diff between the actual and expected test result for each and copying from the former to the latter.
6. (above) requires **manually** updating the 4 jp2 test results AND **manually** updating the `<statistics>` section in 90 test results.

Reply:
I see, I thought you did something to automatically update the entire contents of the expected files. Adding a flag to do this would be quite useful. I might add one this evening.